tests/tensor_search/test_api_exception_handler.py

```diff
@@ -100,3 +107,29 @@ def test_base_exception_handler_unhandled_error(self, mock_api_exception_handler
         self.assertNotIn("This should not be propagated.", converted_error.message)
         self.assertIn("unexpected internal error", converted_error.message)
 
+    async def test_validation_exception_handler_CorrectResponseBody(self):
+        """Ensure that the validation exception handler returns the correct response body when 422 is raised."""
+        # Create a mock request
+        mock_request = MagicMock(spec=Request)
+        # Manually create a RequestValidationError
+        errors = [
+            {
+                "loc": ("body", "name"),
+                "msg": "ensure this value has at most 10 characters",
```

Comment:
Is this message generated by pydantic or is it controlled by us? I was wondering if we should only return this message to the end user as detail. But it lacks the property information.

Reply:
In this test, it is just a manually made one. And we always return the details before.